### PR TITLE
Update Training Script for Vision Models

### PR DESCRIPTION
**Pull request by jacobbieker**
This PR is a subset of #3 and brings together #5 and #6 to train vision reward models. 

This also fixes a bug where the optimizer and scheduler from the training arguments was not being passed or used by the Hugging Face Trainer.

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+)
 from datasets import load_dataset
-from tqdm import tqdm
 import torch
 from torch.optim import AdamW
 from torch.optim.lr_scheduler import CosineAnnealingLR
 
+import argparse
+
+from src_website_quality.utils import get_website_screenshots
```
Comment (amydeng2000) [inline train.py:30]: I think the import needs to adapt to where the utils file live now. I think in the merged PR for website scraping, there's no longer a utils file in /src_website_quality

**Review by amydeng2000 — COMMENTED** [review]
all looks good, left small notes! Will wait for @UYasher to take a final pass

File: train.py [hunk 2]
```diff
@@ -22,93 +45,54 @@ def get_validation_examples():
...
-    else:
-        acc = 0
-
-    result["accuracy"] = acc
-    return result
-
-
-def get_T5_training_settings(_params, _dataset):
+def get_llama_training_settings(
+    params, dataset, gradient_accumulation_steps=4, batch_size=1, num_epochs=1, learning_rate=3e-4
```
[the remark this answers is not in the document]
Comment (UYasher) [reply train.py:48]: I haven't run any kind of hyper-parameter tuning, so all hyperparameters are borrowed from the pre-training setup.
3e-4 is the lr used in pre-training -- see table 2 https://arxiv.org/pdf/2302.13971.pdf
(Also used in llama2 -- https://arxiv.org/pdf/2307.09288.pdf)

**Review by UYasher — APPROVED** [review]
I think this introduces some breaking changes in another repo we use to train the reward model, so a valuable next task might be updating the script which invokes this for training in mosaic.

**Review by UYasher — APPROVED** [review]
